pyproject.toml

```diff
@@ -21,7 +21,7 @@ license = "Apache-2.0"
 [tool.poetry.dependencies]
 python = "^3.7"
 Mopidy = "^3"
-pytube = "^12.1.0"
+pytubefix = "^8.11.0"
```

Comment:
basically this issue and going with that comment: https://github.com/pytube/pytube/issues/2098#issuecomment-2538450069 :woman_shrugging: